apps/src/templates/instructions/ScrollButtons.jsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import Radium from 'radium';
 import color from '../../color';
-import { getOuterHeight } from './utils';
+import { getOuterHeight, scrollBy } from './utils';
+import { addMouseUpTouchEvent } from '../../dom';
 
 const WIDTH = 20;
 const HEIGHT = WIDTH;
 
+const SCROLL_BY = 100;
```

Comment:
maybe some comments on what this and other values mean.. pixels?  when/how often is it called?  etc.